test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -163,15 +165,15 @@ TEST_F(AmetsuchiTest, PeerTest) {
   ASSERT_EQ(peers->size(), 1);
   ASSERT_EQ(peers->at(0)->address(), "192.168.9.1:50051");
 
-  ASSERT_EQ(peers->at(0)->pubkey(), fake_pubkey.hex());
+  ASSERT_EQ(peers->at(0)->pubkey(), fake_pubkey);
 }
 
 TEST_F(AmetsuchiTest, AddSignatoryTest) {
   ASSERT_TRUE(storage);
   auto wsv = storage->getWsvQuery();
 
-  shared_model::crypto::PublicKey pubkey1(std::string(32, '1'));
-  shared_model::crypto::PublicKey pubkey2(std::string(32, '2'));
+  std::string pubkey1{32, '1'};
+  std::string pubkey2{32, '2'};
```

Comment:
```suggestion
  PublicKeyHexStringView pubkey1{"1"};
  PublicKeyHexStringView pubkey2{"2"};
```
and then remove all the explicit ctors of pkeyhexstringview below